keycloak/themes/eddi/login/resources/css/eddi-login.css

```
@@ -0,0 +1,675 @@
+    --eddi-inset-shadow:  inset 0 1px 2px rgba(28, 25, 23, .06);
+    --eddi-card-sheen:    none;
+    --eddi-card-edge:     inset 0 1px 0 0 rgba(255, 255, 255, .9);
+    --eddi-card-shadow:   0 10px 30px -12px rgba(28, 25, 23, .18);
+    --eddi-logo-filter:   invert(1);
+    --pf-v5-global--BoxShadow--lg: 0 6px 18px rgba(28, 25, 23, .10);
+    --pf-v5-global--BoxShadow--xl: 0 12px 32px rgba(28, 25, 23, .14);
+    --eddi-bloom-near:    rgba(245, 158, 11, .10);
+    --eddi-bloom-mid:     rgba(245, 158, 11, .04);
+    --eddi-bloom-far:     transparent;
+    --eddi-vignette:      rgba(120, 113, 108, .10);
+    color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)